spec/models/application_form_spec.rb

```diff
@@ -14,4 +14,22 @@
       expect(application_form.own_and_associated_audits.count).to eq 5
     end
   end
+
+  describe '#update' do
+    it 'updates the application_choices as well' do
```

Comment:
maybe the description ought to be `touches the application_choices as well` or `updates the application_choices updated_at as well`